tests/test_rechunk.py

```diff
@@ -131,6 +242,86 @@ def test_rechunk_dataset(
     assert ds.attrs == dst.attrs
 
 
+@pytest.mark.parametrize("shape", [(100, 50)])
+@pytest.mark.parametrize("source_chunks", [(10, 50), (100, 5)])
+@pytest.mark.parametrize(
+    "target_chunks",
+    [
+        {"x": 20},  # ? Should this rechunk y? Probably not...
+        {"x": 20, "y": 1e5},
```

Comment:
```suggestion
        {"x": 20, "y": 100_000},
```

Better to use integer chunks